server/routes/caseAdmin/handlers/caseload.ts

```diff
@@ -0,0 +1,10 @@
+import { Request, Response } from 'express'
+import CaseAdminCaseloadService from '../../../services/caseAdminCaseloadService'
+
+export default class CaseloadRoutes {
+  constructor(private readonly caseAdminCaseloadService: CaseAdminCaseloadService) {}
+
+  GET = async (req: Request, res: Response): Promise<void> => {
```

Comment:
will integrate with API in following PR.

Reply:
will need a test for this